### DISCUSSION TITLE
הטלת ספק בתשובה של סעיף 8.3 א' וב'

### DISCUSSION THREAD

**Discussion by micha-sherman** [NuclearGandhi/technion_second_brain#8] [טעויות ותיקונים]
אם נפתח את משוואת האנרגיה של הגוף ונציב בה את המידע שמצאנו בעקבות שימור תנע זוויתי נקבל בדיוק את אותה המשוואה תרגיל 8.2 סעיף ב. למשוואה הזאת יש מספר פתרונות ולכן גם המערכת הנל ישנם מספר מצבי שיווי משקל שבהם המרחק מO והמהירות של החלקיים קבועים ובהם מתקיים שימור תנע קווי.
הריי הקנו מהירות הציר הניצב לקפיץ, בהנחה שקבוקע הקפיץ גדול מספיק משום שאין חיכוך המערכת יכולה להמשיך ולהסתובב במהירות קבועה וברדיוס קבוע, אם לא מדובר בR0 כנראה שבאמת יהיו תנודות.
זה מאוד מוזר לי פיזיקאלית אבל נראה שהמתמטיקה מסתדרת (או שאני טועה)

**Comment by micha-sherman** [comment]
![image](https://github.com/user-attachments/assets/e2808ddb-c185-43ae-9408-00a07a35bad8)

**Comment by micha-sherman** [comment]
כשאני חושב על זה עוד, אם יש שימור תנע זוויתי ושימור אנרגיה, מהמשוואה בתחתית הדף נובע כי חייב להיות שימור תנע קווי.
אם לפולינום אין פתרון אז הבעייה עצמה אינה פיזיקלית

**Comment by NuclearGandhi** [comment]
לא כל כך הבנתי באיזה כיוון אתה אומר שיש שימור תנע קווי @micha-sherman ?

**Reply by micha-sherman** [reply]
אני חושב שיש שימור תנע קווי בכיוון הקפיץ (לכיוון מרכז המעגל)

**Reply by NuclearGandhi** [reply]
> אני חושב שיש שימור תנע קווי בכיוון הקפיץ (לכיוון מרכז המעגל)

זהו אבל זה בעייתי לקבוע שימור תנע קווי בכיוון שמשתנה. אומרים לנו גם בתרגולים שאם הכיוון לא קבוע, אי אפשר לומר שהתנע הקווי בכיוונו נשמר.
במקרה הספציפי הזה, אפשר לוודאות גם לומר שאין שימור תנע קווי בכיוון הקפיץ. למשל, בהתחלה אין בכלל למסה תנע קווי בכיוון הקפיץ, אבל לאחר כמה רגעים _יש_ לה בוודאות תנע קווי לכיוון מרכז המעגל, אז אין פה שום תנע קווי שנשמר.

**Comment by micha-sherman** [comment]
אתה צודק, מעבר לזה מערכת הצירים שהגדרתי בעייתית ובהכרח קיימת תאוצה לכיוון מרכז המעגל על מנת לשמור על תנועה מעגלית.
